File: find target indices after sorting.cpp
class Solution {
public:
    vector<int> targetIndices(vector<int>& nums, int target) {
        sort(nums.begin(),nums.end());
        int x=-1,y=-1;
        for(int i=0;i<nums.size();i++)
        {
            if(nums[i]==target)
            {
                x=i;
                break;
            }
        }
        for(int i=nums.size()-1;i>-1;i--)
        {
            if(nums[i]==target)
            {
                y=i;
                break;
            }
            
        }
        vector<int>v;
        if(x<0 and y<0)
            return v;
        for(int i=x;i<=y;i++)
            v.push_back(i);
        return v;
            
        
        
    }
};

File: find target indices after sorting.py
class Solution:
    def targetIndices(self, nums: List[int], target: int) -> List[int]:
        nums.sort()
        x=-1
        y=-1
        for i in range(len(nums)):
            if nums[i]==target:
                x=i
                break
        for i in range(len(nums)-1,-1,-1):
            if nums[i]==target:
                y=i
                break
        t=[]
        if x<0 and y<0:
            return t
        for i in range(x,y+1):
            t+=[i]
        return t
